setup.py

```diff
@@ -20,14 +20,15 @@ def run(self):
         super().run()
         if not self.dry_run:
             try:
-                _run(["tar", "xjf", "htslib.tar.bz2", "-C", "build"])
-                _run(["tar", "xjf", "bcftools.tar.bz2", "-C", "build"])
-                _run(["tar", "xjf", "samtools.tar.bz2", "-C", "build"])
-                _run(["./configure"], cwd="build/htslib")
-                _run(["./configure"], cwd="build/samtools")
```

Comment:
Why don't we need to run configure anymore?

Reply:
The docs recommend it https://github.com/samtools/samtools/blob/9472cc3d06c723ab91b38268421943afa813bbc4/INSTALL#L72 which seems like a good reason to me, but your argument is compelling so if you want to leave it out, fine.

Reply:
Eh, I thought about it and added it back in.